src/Analyzers/CSharp/Tests/UseExpressionBody/UseExpressionBodyForAccessorsAnalyzerTests.cs

```diff
@@ -35,8 +38,10 @@ private static async Task TestWithUseExpressionBody(string code, string fixedCod
                     { CSharpCodeStyleOptions.PreferExpressionBodiedAccessors, ExpressionBodyPreference.WhenPossible  },
                     { CSharpCodeStyleOptions.PreferExpressionBodiedProperties, ExpressionBodyPreference.Never },
                     { CSharpCodeStyleOptions.PreferExpressionBodiedIndexers, ExpressionBodyPreference.Never },
-                }
-            }.RunAsync();
+                },
+            };
+
+            await test.RunAsync();
```

Comment:
nit: consider reverting the changes in this method (not necessary)

Reply:
this was helpful for examining test state :)